mappings/net/minecraft/client/gui/widget/container/LayoutSettings.mapping

```diff
@@ -1,9 +1,48 @@
 CLASS net/minecraft/unmapped/C_pjarmpvz net/minecraft/client/gui/widget/container/LayoutSettings
 	METHOD m_aqbyqkmb create ()Lnet/minecraft/unmapped/C_pjarmpvz;
-	CLASS C_efyrmufb LayoutSettingsImpl
+	METHOD m_awwfcuvd setRightPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_djhyilbe setPadding (II)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 horizontal
+		ARG 2 vertical
+	METHOD m_ewcvnloj setAlignment (FF)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 x
+		ARG 2 y
+	METHOD m_gouapxzs alignVerticallyTop ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_hyvejfbf setPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_ietlmaqj alignVerticallyBottom ()Lnet/minecraft/unmapped/C_pjarmpvz;
```

Comment:
hm, `align<direction><axis>y` might be better

Reply:
mm, I like `alignHorizontallyLeft` over `alignLeftHorizontally`